openstack/cdn/v1/base/requests.go

```diff
@@ -0,0 +1,30 @@
+package base
+
+import (
+	"github.com/rackspace/gophercloud"
+
+	"github.com/racker/perigee"
+)
+
+// Get retrieves the home document, allowing the user to discover the
+// entire API.
+func Get(c *gophercloud.ServiceClient) GetResult {
+	var res GetResult
+	_, res.Err = perigee.Request("GET", getURL(c), perigee.Options{
+		MoreHeaders: c.AuthenticatedHeaders(),
+		Results:     &res.Body,
+		OkCodes:     []int{200},
+	})
+	return res
+}
+
```

Comment:
Do we also want to support the `/health` and `/health/{subsystem}` calls here?

Looking at the responses they seem mostly useful for internal diagnostics, so I could also see the argument for leaving them out. For completeness' sake it might be worth having them, though. "In a later PR" would also be a valid option.